### PR TITLE
move from commonjs to module

### LINKED ISSUE #7: does it support sveltekit?

### PR DESCRIPTION
**Pull request by sapkra**
fixes #7 
I would consider it a potential breaking change so I would release at as 2.0.0

### CODE REVIEW

**Review by ralyodio — APPROVED** [review]

**Comment by NishuGoel** [comment]
Thanks @sapkra 
Do you mind taking care of the failing test as well? See [here](https://github.com/NishuGoel/svelte-i18next/pull/10/checks)

**Comment by sapkra** [comment]
@NishuGoel I updated all the dependencies and it fixed it automatically.

File: tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "strict": true,
     "target": "es6",
-    "module": "commonjs",
```
Comment (NishuGoel) [inline tsconfig.json:5]: So changing the module type makes it compatible with sveltekit, @sapkra?

File: tsconfig.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+    "module": "es6",
```
Comment (NishuGoel) [inline tsconfig.json:5]: Did we test if it now works well with svelte/sveltekit/both used together?

**Review by NishuGoel — APPROVED** [review]

File: tsconfig.json (hunk 1 above)
Comment (sapkra) [reply tsconfig.json:5]: Short answer: yes, we had to use a huge workaround get get it working before. Now we can remove it.

File: tsconfig.json (hunk 2 above)
Comment (sapkra) [reply tsconfig.json:5]: Usually frontend tools like vite are handling modules by themselve to enable on demand file serving. It's currently the best practise to ship npm modules as ES modules.

I personally tested it only with sveltekit but I see non reason why it shouldn't work in other svelte projects.

**Comment by sapkra** [comment]
@NishuGoel Thank you for applying this fix so quickly. I very appreciate it. 🚀

**Pull request merged**